<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopStateBar</class>
 <widget class="QWidget" name="TopStateBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1214</width>
    <height>47</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TopStateBar</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#444444;QLabel{font-family:/pfscr.ttf;}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="20,80,20,140,800,40,200,20,20,20">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>102</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>102</width>
       <height>24</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/img/res/mainwindow/LOGO.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_title">
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
text-align: left;font:24px PingFang SC;</string>
     </property>
     <property name="text">
      <string>安检CT工作站</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_present">
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
text-align: left;font:16px PingFang SC;</string>
     </property>
     <property name="text">
      <string>当前:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#FFFFFF;border-image:url(:/img/res/topstatus/manual.png);border-radius:4px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_process">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#FFFFFF;border-image:url(:/img/res/toolbar/colorc.png);border-radius:4px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_enhence">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#FFFFFF;border-image:url(:/img/res/toolbar/lowprocessc.png);border-radius:4px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_neg">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#FFFFFF;border-image:url(:/img/res/topstatus/normal.png);border-radius:4px</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Scaling">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>24</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>PingFang SC</family>
         <pointsize>-1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#FFFFFF;qproperty-icon:url(:/img/res/mainwindow/normal.png);border-radius:4px;font:16px PingFang SC;</string>
       </property>
       <property name="text">
        <string>1.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="button_help">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{border-image:url(:/img/res/mainwindow/help.png);}
QPushButton::hover{border-image:url(:/img/res/mainwindow/helph.png);}
QPushButton::pressed{border-image:url(:/img/res/mainwindow/helpc.png);}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resource.qrc"/>
  <include location="IPSAPP.qrc"/>
 </resources>
 <connections/>
</ui>
